### ISSUE TITLE
Error opening file "../crypto/rand/randfile.c" while installing Wazuh Manager on Ubuntu 18

### ISSUE THREAD

**Issue by alberpilot** [wazuh/wazuh#3395]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| From 3.7.1 to 3.9.1 | Installation | Manager | Packages and Sources | Ubuntu 18 |

Hello team

  Our colleague @Cerv1 notices that there is an error message installing Wazuh Manager on Ubuntu 18. Below, you'll find the error trace:

Wazuh versión: 3.7.1 to 3.9.1

```
root@c6977e00506b:/# apt-get install wazuh-manager
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Suggested packages:
  expect
The following NEW packages will be installed:
  wazuh-manager
0 upgraded, 1 newly installed, 0 to remove and 1 not upgraded.
Need to get 70.1 MB of archives.
After this operation, 336 MB of additional disk space will be used.
Get:1 https://packages.wazuh.com/3.x/apt stable/main amd64 wazuh-manager amd64 3.9.1-1 [70.1 MB]
Fetched 70.1 MB in 2s (28.9 MB/s)        
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package wazuh-manager.
(Reading database ... 5575 files and directories currently installed.)
Preparing to unpack .../wazuh-manager_3.9.1-1_amd64.deb ...
Unpacking wazuh-manager (3.9.1-1) ...
Setting up wazuh-manager (3.9.1-1) ...
Can't load /root/.rnd into RNG
139871618495552:error:2406F079:random number generator:RAND_load_file:Cannot open file:../crypto/rand/randfile.c:88:Filename=/root/.rnd
Generating a RSA private key
.......+++++
.......................................+++++
writing new private key to '/var/ossec/etc/sslmanager.key'
-----
wazuh-clusterd not running...
wazuh-modulesd not running...
ossec-monitord not running...
ossec-logcollector not running...
ossec-remoted not running...
ossec-syscheckd not running...
ossec-analysisd not running...
ossec-maild not running...
ossec-execd not running...
wazuh-db not running...
ossec-authd not running...
ossec-agentlessd not running...
ossec-integratord not running...
ossec-dbd not running...
ossec-csyslogd not running...
Wazuh v3.9.1 Stopped
Starting Wazuh v3.9.1...
Started ossec-csyslogd...
Started ossec-dbd...
2019/05/27 06:34:53 ossec-integratord: INFO: Remote integrations not configured. Clean exit.
Started ossec-integratord...
Started ossec-agentlessd...
Started ossec-authd...
Started wazuh-db...
Started ossec-execd...
Started ossec-analysisd...
Started ossec-syscheckd...
Started ossec-remoted...
Started ossec-logcollector...
Started ossec-monitord...
Started wazuh-modulesd...
Completed.
```
There is an equivalent error installing from sources. 

Best regards, 
Alberto R.

**Comment by sergiovp** [comment]
This issue is due to a variable defined at `/etc/ssl/openssl.conf`:
```
RANDFILE                = $ENV::HOME/.rnd
```

## Next steps
- Reproduce this issue in v3.6.1.
- Find out which changes were commited from v3.6.1 to v3.7.1.

**Comment by sergiovp** [comment]
## Reproduce the issue in different versions
+ The issue was reproduced in **Wazuh** **v3.9.1**, **v3.6.1** and **v3.3.1** and we got the same error message.
~~~
Can't load /root/.rnd into RNG
139871618495552:error:2406F079:random number generator:RAND_load_file:Cannot open file:../crypto/rand/randfile.c:88:Filename=/root/.rnd
~~~

## Error cause
+ The error message is due the version of **OpenSSL**, concretly **v1.1.1**.
+ This error is fixed in **v1.1.1a**. [Error fixed](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=898470).

## Consequences
+ Despite the error message, **OpenSSL** works well. It generates different keys and certificates values, so it hasn't got important consequences.

## Posible solutions
1. Coment **RANDFILE** variable at the beginig of `/etc/ssl/openssl.conf` file.
2. Ignore error message.

## Conclusion
Considering that it doesn't affect to our system's security, we could **ignore the error message**.
